src/server/controllers/item.js

```diff
@@ -3,9 +3,10 @@ const filter = require('../common/utilities');
 
 // Create new item
 const itemCreate = (req, resp) => {
-  const { author, isOrdered, name } = req.body;
+  const { author, authorId, isOrdered, name } = req.body;
```

Comment:
new variable is authorId, maybe it would be good to change the name of the author variable to the authorName, this would be more self-documenting